pkg/cli/clients/management_test.go

```diff
@@ -1355,3 +1353,158 @@ func setCapture(ctx context.Context, response *http.Response) {
 func testCapture(ctx context.Context, capture **http.Response) context.Context {
 	return context.WithValue(ctx, holder{}, &holder{capture})
 }
+
+func Test_isResourceInEnvironment(t *testing.T) {
+	// Setup test cases
+	type testCase struct {
+		name          string
+		resource      generated.GenericResource
+		environmentID string
+		expected      bool
+	}
+
+	fullEnvID := "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Applications.Core/environments/env1"
```

Comment:
Is this a valid environment ID? This looks more like the provider scope to me, but maybe there's a scenario in which this becomes an ID?

Reply:
Fixed